alns/ALNS.py

```diff
@@ -124,6 +128,11 @@ class of vehicle routing problems with backhauls. *European Journal of
         d_weights = np.ones_like(self.destroy_operators, dtype=np.float16)
         r_weights = np.ones_like(self.repair_operators, dtype=np.float16)
 
+        statistics = Statistics()
+
+        if collect_stats:
```

Comment:
I don't really like this duplicate statement (see below). Perhaps this can be brought into the loop?